docs/_documentations/remote-deploying-codewind.md

```diff
@@ -120,12 +159,12 @@ NAME       USERNAME   NAMESPACE   AGE    KEYCLOAK   REGISTRATION   ACCESSURL
 devex001   jane       codewind    119m   devex001   Complete       https://codewind-gatekeeper-devex001.<ingress-domain>
 ```
 
-## 5. Removing a Codewind instance
-
-To remove a Codewind instance, see [Removing a Codewind instance](https://github.com/eclipse/codewind-operator/blob/master/README.md#removing-a-codewind-instance) in the codewind-operator readme.
-
 ## Next steps
 
 You have now finished installing the Codewind operator, and you have deployed a Codewind instance.
 
-In the next topic, you will learn how to [use Codewind remotely](./remote-codewind-overview.html).
+In the next topic, you will learn how to [use Codewind remotely](./remote-codewind-overview.html). 
+
+You will configure your IDE to connect to Codewind in the cloud, for more information, see [Connecting your VS Code to remote Codewind](remotedeploy-vscode.html) or [Connecting your Eclipse to remote codewind](./remotedeploy-eclipse.html).
```

Comment:
The clause beginning with "for more info..." should be a separate sentence. As it stands, it is a comma splice. If you want to keep the clauses connected, you need a comma plus a conjunction.

"You will configure your IDE to connect to Codewind in the cloud. For more information, see..."
or
"You will configure your IDE to connect to Codewind in the cloud, and for more information, see..."

Reply:
First suggestion adopted, comma splice resolved.